weather.js

```diff
@@ -0,0 +1,70 @@
+const axios = require("axios");
+const dotenv = require('dotenv');
+dotenv.config();
+
+const apiURLPrefix = "http://api.openweathermap.org/";
+
+/**
+ * Runs a conditional to distinguish if Zipcode or address string was entered.
+ * @param {string|number}input
+ * @returns {Promise<void>}
+ */
+async function getWeather(input) {
+    if (isNaN(input)) {
+        const data = await getCoordinates(input);
```

Comment:
Variable names like 'data' are not good.
How about something like `coordinates` ?

Reply:
Good catch, much better name.